package_manifest.yaml

```diff
@@ -131,10 +131,6 @@ foreman_scl_packages:
     rubygem-locale: {}
     rubygem-memoist: {}
     rubygem-multipart-post: {}
-    rubygem-mysql2:
```

Comment:
This is currently still needed for the `smart_proxy_dns_powerdns` package but I've deprecated the direct DB versions so I can release a new version that just supports the API. https://github.com/theforeman/smart_proxy_dns_powerdns/commit/deec2285488109c66b78c9ebae62456098602ee7 was already merged. Since it has no extra dependencies I also considered merging it into `smart-proxy` itself but I'll first release 0.5.0.